packages/aws-cdk/package.json

```diff
@@ -138,7 +138,7 @@
     "@smithy/util-waiter": "3.1.6",
     "archiver": "^5.3.2",
     "camelcase": "^6.3.0",
-    "cdk-assets": "^3.0.0-rc.48",
+    "cdk-assets": "^3.0.0-rc.111",
```

Comment:
This versions contains the schema bump: https://github.com/cdklabs/cdk-assets/releases/tag/v3.0.0-rc.111